CellDataCoupler/Classes/EmptyCell/EmptyCell.swift

```diff
@@ -0,0 +1,18 @@
+//
+//  EmptyCell.swift
+//  CellDataCoupler
+//
+//  Created by Jangle's MacBook Pro on 10/29/20.
+//
+
+import UIKit
+
+class EmptyCell: BaseReusableCell, BundledIdentifiableView {
```

Comment:
The need for this seems dependent on the fact that we aren't returning nil for getItem when an item doesn't exist.
Fixing the issue with getItem will likely remove the need for this.

Reply:
Check out the long-winded reply above

Reply:
After talking, we'll close this in favor of using a zero-height empty cell if a coupler isn't successfully loaded.